src/platforms/f4discovery/platform.c

```diff
@@ -39,6 +40,11 @@
 jmp_buf fatal_error_jmpbuf;
 extern uint32_t _ebss; // NOLINT(bugprone-reserved-identifier,cert-dcl37-c,cert-dcl51-cpp)
 
+int platform_hwversion(void)
```

Comment:
Is there a particular reason for removing this from the platform header? This will impair the optimiser's ability to optimise this function out to just a value in all call sites